src/overlays/actors/ovl_En_Scopenuts/z_en_scopenuts.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_en_scopenuts.c
  * Overlay: ovl_En_Scopenuts
- * Description: Astral Observatory - Business Scrub in Telescope
+ * Description: Business Scrub that sells you a Heart Piece (viewable from Telescope)
```

Comment:
Is the `(viewable from Telescope)` specifying that it's only this actor as viewed by the telescope i.e. the one you talk to in the grotto is a different actor? Or is the purpose to just clarify which business scrub it is and both what you see in the telescope and the one you talk to in the grotto are the same

Reply:
It's the latter. I think I'll just remove the `(viewable from Telescope)` thing because while it was intended to clarify things, it actually makes it more confusing.